examples/40_paper/2015_neurips_feurer_example.py

```diff
@@ -10,9 +10,80 @@
 ~~~~~~~~~~~
 
 | Efficient and Robust Automated Machine Learning
-| Matthias Feurer, Aaron Klein, Katharina Eggensperger, Jost Springenberg, Manuel Blum and Frank Hutter
+| Matthias Feurer, Aaron Klein, Katharina Eggensperger, Jost Springenberg, Manuel Blum and Frank Hutter  # noqa F401
```

Comment:
@mfeurer wrong usage of # noqa F401 in the text ? It is not interpreted as a comment.
Maybe you meant here:
https://github.com/openml/openml-python/blob/5a2830cc494dbffe93fb324aa7eb98b8bf3f0b33/examples/40_paper/2015_neurips_feurer_example.py#L18